tests/scripts/check_names.py

```diff
@@ -444,8 +444,11 @@ def parse_enum_consts(self, include, exclude=None):
                     # Match typedefs and brackets only when they are at the
                     # beginning of the line -- if they are indented, they might
                     # be sub-structures within structs, etc.
+                    optional_c_identifier = r"([_a-zA-Z][_a-zA-Z0-9]*)?"
                     if (state == states.OUTSIDE_KEYWORD and
-                            re.search(r"^(typedef +)?enum +{", line)):
+                            re.search(r"^(typedef +)?enum " + \
+                                    optional_c_identifier + \
+                                    r" *{", line)):
```

Comment:
might the `{` ever be on a new line?

Reply:
Yep, this is covered by the other cases in the state machine - if no brace is found within the line, the state is set to `IN_BETWEEN` (see [line 453](https://github.com/Mbed-TLS/mbedtls/pull/6810/files#diff-14b1a15e2eedf4a429d6540d6b8531379bf9e019d408590824abb6e7cec32530R453)).